metabox/metabox/core/machine.py

```diff
@@ -165,19 +166,25 @@ def get_late_setup(self):
         """
         return []
 
-    def start_remote(self, host, launcher, interactive=False, timeout=0):
+    def start_remote(self, host, launcher, cmd="remote", interactive=False, timeout=0):
+        """
+        Start remote
+        Note: This method will inject host and launcher at the end of cmd, if this is
+              undesirable, use run_cmd
+        """
+        if not cmd:
+            cmd = "remote"
         assert self.config.role == "remote"
+        cmd_arr = shlex.split(cmd)
```

Comment:
you introduced the cmd with this patch, but here you change the type of it. If you want an list, just make it so.
Also what's with the piratey arr? Is it short for array, which kinda this is not.

Reply:
I want them all to have the same type, the original one is a str and I did not change it, so it remain str everywhere for uniformity. The arr stands for array, I can change it to lst given that there is an array module, but I don't think that anyone reading it would have any doubt on what the type is, I just didnt want to alias the one above